<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassportHeaderFormControl</class>
 <widget class="QWidget" name="PassportHeaderFormControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="sampleLabel">
       <property name="text">
        <string>Sample</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="FluidLabel">
       <property name="text">
        <string>Fluid</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="sampleComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="fluidComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commentLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sampleComboBox</tabstop>
  <tabstop>fluidComboBox</tabstop>
  <tabstop>commentLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
